package com.jcirmodelsquad.tcjcir.vehicles.rollingstock.tender;

import net.minecraft.entity.item.EntityMinecart;
import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.inventory.IInventory;
import net.minecraft.item.ItemStack;
import net.minecraft.nbt.NBTTagCompound;
import net.minecraft.nbt.NBTTagList;
import net.minecraft.world.World;
import net.minecraftforge.common.util.Constants;
import net.minecraftforge.fluids.FluidRegistry;
import train.common.Traincraft;
import train.common.api.LiquidManager;
import train.common.api.Tender;
import train.common.library.EnumTrains;
import train.common.library.GuiIDs;

public class TenderNP_11C extends Tender  {

	
	

	public TenderNP_11C(World world) {
		super(world, FluidRegistry.WATER, 0, EnumTrains.NP_11C_Tender.getTankCapacity(), LiquidManager.WATER_FILTER);
		
	}

	@Override
	public String getInventoryName() {
		return "NP 11C/12C Tender(s)";
	}

	

	
	@Override
	public float getOptimalDistance(EntityMinecart cart) {
		return 1.3F;
	}

	
}